docs/backend-crd-reference.md

```diff
@@ -0,0 +1,151 @@
+# Backend Custom Resource Reference
+
+## Full CR Example
```

Comment:
I think it will be also great to a have a simpler deployment, just to show how deploy with just the required fields and minor changes from defaults.

Reply:
Normally I put a simple but production ready CR on the default path inside deploy/crds/. Actually having there a good real example (not invented images..), is really important, because it is used by OLM/CSV (automatically) documentation as a real example that need to contain correct data to be able to deploy it.

Reply:
Sounds good too, but the full example in the documentation can be overwhelming. Maybe we can have an intro section with the simplified cr as example,  then the full CR with all set and finally the spec table.

Reply:
Fixed https://github.com/3scale/saas-operator/pull/6/commits/141bae76019d0b86fcc1dcd3064d2ee44d599ebf

Reply:
Awesome!